docs/deploy.md

```diff
@@ -156,19 +168,25 @@ Copy the Maven project package plus dependencies configured with the `copyDepend
                             <appsDirectory>apps</appsDirectory>
                             <stripVersion>true</stripVersion>
                             <copyDependencies>
+                                <stripVersion>true</stripVersion>
+                                <!-- copies the commons-logging:commons-logging:1.0.4 dependency plus transitive dependencies
+                                     to the default location lib/global folder of the target server and strips the version. -->
                                 <dependency>
-                                    <!-- copies the commons-logging:commons-logging:1.0.4 dependency plus transitive 
-                                         dependencies to the default location ${server.config.dir}/lib/global -->
-                                    <filter>commons-logging</filter>
+                                    <groupId>commons-logging</groupId>
+                                    <artifactId>commons-logging</artifactId>
+                                    <version>1.0.4</version>
                                 </dependency>
-                                <dependency>
-                                    <!-- copies the org.apache.derby:derby:10.15.2.0 and org.apache.derby:derbyclient:10.15.2.0 
-                                         dependencies plus transitive dependencies to the specified location 
-                                         ${server.config.dir}/lib/global/derby and strips the version during the copy. -->
-                                    <filter>org.apache.derby:derby*</filter>
+                                <!-- copies the org.apache.derby:derby:10.15.2.0 and org.apache.derby:derbyclient:10.15.2.0 
+                                     dependencies plus transitive dependencies to the lib/global/derby folder of the target 
+                                     server and strips the version during the copy. -->
+                                <dependencyGroup>
                                     <location>lib/global/derby</location>
-                                    <stripVersion>true</stripVersion>
-                                </dependency>
+                                    <dependency>
+                                        <coordinate>org.apache.derby</coordinate>
+                                        <groupId>org.apache.derby</groupId>
+                                        <artifactId>derby*</artifactId>
```

Comment:
In this example, this is the same as omitting the <artifactId>... want to say that??

Reply:
I'm trying to give an example of using the wildcard.